### DISCUSSION TITLE
Error installing Whisper

### DISCUSSION THREAD

**Discussion by EmmaJane1** [openai/whisper#1434] [Q&A]
Getting the following error message when attempting to install whisper. 
Pycharm is installed and up to date.
ffmpeg has been already installed.
Python version:3.11, running on Windows
I'm not familiar with it, please tell me what do I need to do before using it?

Error message:
_C:\whisper\whisper\venv\Scripts\python.exe C:\whisper\whisper\test.py 
C:\whisper\whisper\whisper\timing.py:57: NumbaDeprecationWarning: The 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.
  @numba.jit
Traceback (most recent call last):
  File "C:\whisper\whisper\test.py", line 2, in <module>
    whisper_model = whisper.load_model("large")
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\whisper\whisper\whisper\__init__.py", line 131, in load_model
    checkpoint_file = _download(_MODELS[name], download_root, in_memory)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\whisper\whisper\whisper\__init__.py", line 59, in _download
    model_bytes = f.read()
                  ^^^^^^^^
MemoryError_

And here is the code of test.py 

_import whisper
whisper_model = whisper.load_model("large")
result = whisper_model.transcribe(r"E:\Python\OpenPROS\Whisper\whisper-20230314\venv\Lib\site-packages\sympy\physics\optics\tests\test.wav")
print(", ".join([i["text"] for i in result["segments"] if i is not None]))_

Besides, I run setup.py, but get error message again.

_C:\Python3.11\python.exe C:\whisper\whisper\setup.py 
C:\Python3.11\Lib\site-packages\setuptools\_distutils\dist.py:264: UserWarning: Unknown distribution option: 'readme'
  warnings.warn(msg)
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: no commands supplied

Many thanks for your reply.

**Comment by phineas-pta** [comment]
what installation command are you using? you have to use `pip`
